CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [10.4.0] - 2018-04-13
+### Added
+- added restricted scopes for thunderbird
+- update generating keys to support non-`cltsign` user names
```

Comment:
I changed the output filenames of `create_gpg_keys` and updated the docs to not hardcode cltsign.